purchase_exception/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Purchase Exception
+==================
+
+This module allows you attach several customizable exceptions to your
+purchase order in a way that you can filter orders by exceptions type and fix them.
+
+This is especially useful in an scenario for mass purchases order import, because it's likely some orders have
+errors when you import them (like product not found in Odoo, wrong line
+format etc.)
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/167/12.0
```

Comment:
Looks like its copied from sale exception since past version. Real id is 142, not 167.